src/scripts/meet.js

```diff
@@ -23,8 +24,19 @@ setInterval(function () {
     )[0];
     joinButton.click();
     console.log('Clicked join button');
+    
   } catch (err) {
     console.log(err);
   }
+  try {
```

Comment:
Just add a check if the meeting is being recorded or not because if it is not being recorded, there is no point in running this script.